includes/discovery/sensors/state/fortigate.inc.php

```diff
@@ -69,6 +69,9 @@
 
     // Per-node loop
     foreach ($haStatsEntries as $index => $entry) {
+        // Get current value (for issue #16544)
+        $sensor_value = snmp_get($device, $fgHaStatsSyncStatus_txt . '.' . $index, '-OeQv', 'FORTINET-FORTIGATE-MIB');
```

Comment:
Looks like we do nothing with `$fgHaStatsSyncStatus_txt` before hand other than set it and unset it so you can safely append the `$index` to the variable a few lines above.